tests/test_storage.py

```diff
@@ -681,10 +509,10 @@ def test_object_acl(self):
 
     def test_storage_class(self):
         """Can specify a storage class for GCS objects"""
-        settings = dict(self.settings)
+        settings = self.settings.copy()
         settings["storage.storage_class"] = "COLDLINE"
-        kwargs = GoogleCloudStorage.configure(settings)
-        storage = GoogleCloudStorage(MagicMock(), **kwargs)
+        storage_func = get_storage_impl(self.settings)
+        storage = storage_func(MagicMock())
```

Comment:
might it be because I switched to `get_storage_impl`? coveralls should still report hits on the actual class though right?